src/pages/community/index.mdx

```diff
@@ -34,6 +34,8 @@ discussion, questions, and announcements.
 meetups in cities all over the world.
 * **StackOverflow:** While the user mailing list is the primary resource for asking questions, if you prefer
 StackOverflow, make sure to tag your question with `druid` or `apache-druid`.
+* **Struct:** Check out [Apache Druid on Struct](https://druid.struct.ai/kb) for an AI-generated knowledge base from Apache Druid Slack threads.
```

Comment:
can we add a note somewhere here that Slack has 90 days of history and linen and struct is where old slack threads are archived at. Also, please move linen over Struct.